src/gpu/intel/jit/gemm/gen_gemm.hpp

```diff
@@ -567,6 +576,24 @@ struct gen_gemm_t : public gpu_gemm_t {
             auto dotrans = batch ? acb : ba;
             auto notrans = batch ? abc : ab;
 
+            auto byte_align_md = [&](memory_desc_t &md) {
+                auto dim = md.dims[md.ndims - 1];
+                dnnl::impl::dims_t dims;
+                dnnl::impl::utils::array_copy(dims, md.dims, md.ndims);
+                auto kernel_type = convert_dnnl_to_kernel_type(md.data_type);
+                bool sub_byte = kernel_type.isInt4() || kernel_type.isFP4();
+                dim_t cache_line_elems
+                        = sub_byte ? 64 * 2 : 64 / kernel_type.size();
```

Comment:
This works for all data types:
```suggestion
                dim_t cache_line_elems = 64 / kernel_type.size();
```

Reply:
`kernel_type.size() `returns `stub()` for sub byte data types.

Reply:
Sorry, that should have been `64 / kernel_type`.